lib/src/content_hash.rs

```diff
@@ -2,6 +2,7 @@
 
 use blake2::Blake2b512;
 use itertools::Itertools as _;
+pub use proc_macros::ContentHash;
```

Comment:
nit: `pub(crate)` if this isn't supposed to be used outside the jj-lib crate?

Reply:
Resolved for now. Depending on what we decide in the other thread we may make this public outside of the crate as well.